src/shmem_internal.h

```diff
@@ -44,6 +44,13 @@ extern long shmem_internal_data_length;
 /* Note: must be accompanied by shmem_internal_my_pe in arguments */
 #define RAISE_PE_PREFIX "[%04d]        "
 
+
+/* Function prototype for standard routines with the default context: */
+#define SHMEM_FUNC_PROTOTYPE(FUNCNAME,  ...)        \
+  shmem_##FUNCNAME(__VA_ARGS__) {                   \
+  const shmem_ctx_t ctx = SHMEM_CTX_DEFAULT;
+
+
```

Comment:
I'd prefer to move this into the .c files to where it is used (and has limited lifespan between define/undefine), rather than defining it at a top level across all files.

Reply:
Yeah I think so too.  I originally had this, but thought it was slightly weird to have it duplicated in `atomic_c.c4` and `data_c.c4`...